packages/eas-cli/src/build/errors.ts

```diff
@@ -0,0 +1,53 @@
+export class EasCommandError extends Error {
```

Comment:
This error is not build-specific, move it somewhere close to code that is handling the error, probably in commandUtils directory